### ISSUE TITLE
Can aihwkit process massive amounts of data？

### ISSUE THREAD

**Issue by FrankWu1998** [IBM/aihwkit#300]
## Description and motivation
Hi, 
Is the tensor created with PyTorch first placed in the main memory and then transferred to the PIM chip?
When the neural network model needs to process a large amount of data, aihwkit should first put on the data the main memory. Therefore, I wonder whether aihwkit can simulate the data transfer between the main memory and the PIM chip.

**Comment by maljoras** [comment]
Hi @FrankWu1998 ,
many thanks for the question!  However, I am not sure whether I understood your question entirely. Note that AIHWKIT simulates the noise sources of analog training and inference for DNN workloads. It is *not* a tool that estimates run-times or energy performance, but rather is focused on the expected accuracy with given noise sources when using a a specific material or update algorithm.  Thus, to estimate  accuracy it is irrelevant how the digital memory is used.  

Note sure whether that answers your question.

**Comment by FrankWu1998** [comment]
Hi @maljoras ,
Thank you very much for your reply. Your answer just solved my confusion about aihwkit. I had always mistakenly thought that it can simulate the latency of DNN workloads.
